### PR TITLE
Deploy to AWS application repository

### PR DESCRIPTION
**Pull request by zaro0508**
* Setup metadata for the AWS serverless app repository[1]
* Make CI Publish lambda to the repository on every update
* Once pubished it will be available in the global AWS serverless repo[2]

[1] https://docs.aws.amazon.com/serverlessrepo/latest/devguide/serverlessrepo-publishing-applications.html
[2] https://serverlessrepo.aws.amazon.com/applications

### CODE REVIEW

File: template.yaml [hunk 1]
```diff
@@ -3,6 +3,20 @@ Transform: AWS::Serverless-2016-10-31
...
+  AWS::ServerlessRepo::Application:
+    Name: "lambda-mongo-health-check"
+    Description: "This is a Lamba to check the health of a Mongo database running on an EC2"
+    Author: "zaro0508"
+    SpdxLicenseId: "Apache-2.0"
+    # paths are relative to .aws-sam/build directory
+    LicenseUrl: "LICENSE"
+    ReadmeUrl: "README.md"
+    Labels: ["serverless", "custom-resource", "mongo", "database"]
+    HomePageUrl: "https://github.com/Sage-Bionetworks-IT/cfn-cr-s3-notification-configuration"
```
Comment (xschildw) [inline template.yaml]: Should be 'https://github.com/Sage-Bionetworks-IT/lambda-mongo-health-check' ?

File: template.yaml [hunk 2]
```diff
@@ -3,6 +3,20 @@ Transform: AWS::Serverless-2016-10-31
...
+    SemanticVersion: "0.0.1"
+    SourceCodeUrl: "https://github.com/Sage-Bionetworks-IT/cfn-cr-s3-notification-configuration/tree/0.0.1"
+
```
Comment (xschildw) [inline template.yaml:19]: Same?

File: template.yaml (hunk 1 above)
Comment (zaro0508) [reply template.yaml]: fixed

File: template.yaml (hunk 2 above)
Comment (zaro0508) [reply template.yaml:19]: fix

**Review by xschildw — APPROVED** [review]
Should there be a branch 0.0.1 to match the SourceCodeUrl?

**Comment by zaro0508** [comment]
> Should there be a branch 0.0.1 to match the SourceCodeUrl?

yes, that will be created after this PR merges

File: template.yaml [hunk 3]
```diff
@@ -3,6 +3,20 @@ Transform: AWS::Serverless-2016-10-31
 Description: >
   Lambda to check Mongo database health
 
+Metadata:
+  AWS::ServerlessRepo::Application:
+    Name: "lambda-mongo-health-check"
+    Description: "This is a Lamba to check the health of a Mongo database running on an EC2"
```
Comment (brucehoff) [inline template.yaml]: LambDa

**Review by brucehoff — CHANGES_REQUESTED** [review]

File: template.yaml (hunk 3 above)
Comment (zaro0508) [reply template.yaml]: fixed

**Pull request merged**